src/v2/Components/NavBar/Menus/MobileNavMenu/MobileNavMenu.tsx

```diff
@@ -291,7 +295,7 @@ const AuthenticateLinks: React.FC = () => {
     <Box>
       <Separator my={1} color={color("black10")} />
       <MobileLink href={authLink(ModalType.signup)}>Sign Up</MobileLink>
-      <MobileLink href={authLink(ModalType.login)}>Login</MobileLink>
+      <MobileLink href={authLink(ModalType.login)}>Log In</MobileLink>
```

Comment:
The casing should be consistent with desktop?

![](http://static.damonzucconi.com/_capture/cwzEw8za9Beo.png)

Reply:
Right - I took the case from `Sign Up` in the line above, but I'll change them both to match desktop.